R/rules-line_break.R

```diff
@@ -42,8 +42,51 @@ remove_line_break_before_round_closing <- function(pd) {
   pd
 }
 
+#' @importFrom rlang seq2
 add_line_break_after_pipe <- function(pd) {
   is_special <- pd$token %in% c("SPECIAL-PIPE")
   pd$lag_newlines[lag(is_special)] <- 1L
   pd
 }
+
+
+#' Set line break for multi-line function calls
+#' @param pd A parse table.
+#' @param except A character vector with tokens before "'('" that do not
+#' @name set_line_break_if_call_is_multi_line
+#' @importFrom rlang seq2
+NULL
+
+#' @describeIn set_line_break_if_call_is_multi_line Sets line break after
+#'   opening parenthesis.
+set_line_break_after_opening_if_call_is_multi_line <-
+  function(pd,
+           except_token = NULL) {
+  if (!is_function_call(pd)) return(pd)
+  npd <- nrow(pd)
+  if (npd == 3) {
```

Comment:
Uups, no error then, sorry. Buth by dropping the if condition, we cannot solve the problem because if `npd = 3`, and
```r
call(
)
```
we want to remove the line break before the closing parent, which we can't using `seq2(a, b)` with `a>b`. Also, my suggestion to use `seq(3, npd)` does not make sense since 
```r
call(2
)
```
would then be classified as multi-line, which I think does not make sense. We should not take the closing parenthesis into account. What do you think? Not sure whether there is another way to do that then.